jekyll/_cci2/hello-world-windows.md

```diff
@@ -336,10 +336,19 @@ Above, we start by declaring that we will use version `2.1` of CircleCI, giving
 
 ```yaml
 orbs:
-  win: circleci/windows@2.2.0
+  win: circleci/windows@2.4.0
 ```
 
-Next, we declare orbs that we will be using in our build. We will only use the [windows orb](https://circleci.com/developer/orbs/orb/circleci/windows) to help us get started.
+Next, we declare orbs that we will be using in our build. We will only use the [windows orb](https://circleci.com/developer/orbs/orb/circleci/windows) to help us get started. This example uses the 2.4.0 version of the orb, but you may consider using a more recent verion.
```

Comment:
Same question, is there a specific reason we continue to use an older version of the orb in some examples?